tests/testSelectSdssImages.py

```diff
@@ -28,9 +28,25 @@
 import lsst.afw.coord as afwCoord
 import lsst.afw.geom as afwGeom
 from lsst.obs.sdss.selectSdssImages import SelectSdssImagesTask
+import lsst.log
+
+logger = lsst.log.Log.getLogger("obs_sdss.testSelectFluxMag0")
 
 Database = "test_select_sdss_images"
 
+config = SelectSdssImagesTask.ConfigClass()
+
+# Some of the tests require loading SDSS images from "lsst-db.ncsa.illinois.edu" and
+# require a login name and password. If the test is unable to connect to the external data,
+# some of the tests are skipped.
+noConnection = False
+try:
+    DbAuth.username(config.host, str(config.port)),
+except Exception as e:
+    logger.warn("Did not find host={0}, port={1} in your db-auth file; \nWarning generated: {2} ".format(
```

Comment:
For test output I think it makes more sense for these messages to go to stderr so that they can be caught by the test harness and displayed on Jenkins.